libraries/AP_NavEKF3/AP_NavEKF3_Measurements.cpp

```diff
@@ -1040,12 +1040,13 @@ void NavEKF3_core::writeExtNavVelData(const Vector3f &vel, float err, uint32_t t
     timeStamp_ms -= localFilterTimeStep_ms/2;
     // Prevent time delay exceeding age of oldest IMU data in the buffer
     timeStamp_ms = MAX(timeStamp_ms,imuDataDelayed.time_ms);
-    const ext_nav_vel_elements extNavVelNew {
-        vel,
-        err,
-        timeStamp_ms,
-        false
-    };
+
+    ext_nav_vel_elements extNavVelNew;
+    extNavVelNew.time_ms = timeStamp_ms;
+    extNavVelNew.vel = vel;
+    extNavVelNew.err = err;
+    extNavVelNew.corrected = false;
```

Comment:
Should be able to use designator initialisation... hopefully...

Reply:
nope, doesn't work